webapp/src/desktop_notifications.ts

```diff
@@ -21,6 +33,20 @@ export function desktopNotificationHandler(store: Store, post: Post, channel: Ch
             ringingEnabled(store.getState())) {
             return {args: {...args, notify: false}};
         }
+
+        // Do not notify for a call's thread if the user is currently in that call...
+        if (channelIDForCurrentCall(store.getState()) === post.channel_id &&
+            callThreadIDForCurrentCall(store.getState()) === post.root_id) {
```

Comment:
Would this work as expected on Desktop? On the main app we wouldn't be on a call redux state wise I think.

Reply:
Yep, looks like we get call state from `loadCallState` (from ws event `call_state`) and from the `handleCallStart` (ws event `call_start`).

Reply:
Okay, I was mostly worried about the channel ID for current call being set but `DESKTOP_WIDGET_CONNECTED` should take care of it already. We are good :+1:

Reply:
Ah yah, that too. Hah, funny enough, this took a bit longer than expected because I originally added the rootID to that desktop connected message (all through the stack), was about to push it, and then saw the redux state and thought, "there's an easier way to do this". 🤕